components/ldap.rst

```diff
@@ -37,15 +37,11 @@ using the following options:
 ``version``
     The version of the LDAP protocol to use
 
-``useSsl``
-    Whether or not to secure the connection using SSL
-
-``useStartTls``
+``encryption``
     Whether or not to secure the connection using StartTLS
```

Comment:
We need to update this sentence too as the possible values are `ssl`, `tls`, and `none`.